backend/src/packages/users/user.repository.ts

```diff
@@ -42,6 +52,34 @@ class UserRepository implements Repository {
   public delete(): ReturnType<Repository['delete']> {
     return Promise.resolve(true);
   }
+
+  public async findByEmail(email: string): Promise<UserEntity | null> {
+    const user = await this.userModel.query().findOne({ email }).execute();
```

Comment:
```suggestion
    const user = await this.userModel.query().findOne({ email });
```
I think we don't need it. Please, correct me if I'm wrong

Reply:
Changed.